www/src/css/_home.scss

```diff
@@ -0,0 +1,90 @@
+@import './_var';
+
+.home {
+  .grid-body {
+    padding-top: 0;
+  }
+
+  .grid-body-header {
+    background: #2e5e82;
+    border-top: solid 5px #ffffff;
+    color: #fff;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    line-height: 1.5;
+    padding: $padding;
+    margin-top: 2em;
+
+    @media (min-width: $breakpoint-m) {
+      margin: 0;
+    }
+    background-image: url("data:image/svg+xml, %3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 640 512' title='mountain' class='logo' fill='white'%3E%3Cpath d='M634.92 462.7l-288-448C341.03 5.54 330.89 0 320 0s-21.03 5.54-26.92 14.7l-288 448a32.001 32.001 0 0 0-1.17 32.64A32.004 32.004 0 0 0 32 512h576c11.71 0 22.48-6.39 28.09-16.67a31.983 31.983 0 0 0-1.17-32.63zM320 91.18L405.39 224H320l-64 64-38.06-38.06L320 91.18z' /%3E%3C/svg%3E");
+    background-size: 50%;
+    min-height: 50vh;
+    background-position: calc(100% + 100px) calc(100% + 40px);
+    background-repeat: no-repeat;
+    a {
+      color: white;
+    }
+
+    > svg {
+      margin: auto;
+      display: block;
+      opacity: 0.9;
+    }
+
+    .logo {
+      width: 100%;
+      position: absolute;
+      right: 0px;
+    }
+    .logo path {
+      fill: #fff;
+    }
+
+    .section {
+      padding: $padding * 2;
+    }
+  }
+
+  .home-grid {
+    width: 100%;
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    grid-gap: 1rem;
+    padding: $padding;
+
+    @media (min-width: $breakpoint-m) {
+      grid-template-columns: repeat(3, 1fr);
+    }
+
+    .section {
+      padding: 5rem 1rem;
+      text-align: center;
+
+      border: 12px solid #2e5e82;
+      border-radius: 40px;
```

Comment:
also played around with some 90 angles and box shadows: 

<img width="450" alt="Screen Shot 2020-11-17 at 8 46 14 PM" src="https://user-images.githubusercontent.com/622227/99484785-01bb3f00-2916-11eb-8a2a-9b088499f57c.png">